packages/editor/src/plugins/image/image-selection-screen.tsx

```diff
@@ -0,0 +1,51 @@
+import { isTempFile } from '@editor/plugin'
+import { faMagnifyingGlass } from '@fortawesome/free-solid-svg-icons'
+import React from 'react'
+
+import type { ImageProps } from '.'
+import { UploadButton } from './controls/upload-button'
+import { FaIcon } from '@/components/fa-icon'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+
+export function ImageSelectionScreen(props: ImageProps) {
+  const editorStrings = useEditorStrings()
+  const { state } = props
+  const { src } = state
+
+  const imageStrings = editorStrings.plugins.image
+  const disableFileUpload = props.config.disableFileUpload // HACK: Temporary solution to make image plugin available in Moodle & Chancenwerk integration with file upload disabled.
+
+  const placeholder = !isTempFile(src.value)
+    ? imageStrings.placeholderEmpty
+    : !src.value.failed
+      ? imageStrings.placeholderUploading
+      : imageStrings.placeholderFailed
+
+  return (
+    <div className="mx-auto rounded-md bg-yellow-50 p-8 shadow-md">
+      <div className="mx-auto my-8 w-[60%]">
+        <UploadButton {...props} />
+        {/* TODO: Before this goes to production, hide search button until functionality is added */}
```

Comment:
You could do this already, using the `isProduction` helper.

Reply:
Thanks, done